app/src/atoms/buttons/TouchControlButton.tsx

```diff
@@ -0,0 +1,102 @@
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+import {
+  BORDERS,
+  Btn,
+  COLORS,
+  CURSOR_DEFAULT,
+  SPACING,
+  StyledText,
+} from '@opentrons/components'
+
+import { getIsOnDevice } from '/app/redux/config'
+
+import type { MouseEventHandler } from 'react'
+
+interface TouchControlProps {
+  text1: string
```

Comment:
```suggestion
  title: string
```
Nit, this is the semantic naming convention we use for props such as this one.